### PR TITLE
Optimize ignore apps list view

### PR DESCRIPTION
**Pull request by phlpsong**
1. Some app's name is different in Applications with Ignore app list(`VidHub` named `MediaCenter` in ignore app)
2. Chose the same app multiple times will have duplicate items
3. Icon view may shown cached image if the app is removed(last item in below screenshot)

SCR-20240327-ujze

### CODE REVIEW

**Comment by phlpsong** [comment]
It looks like have an issue with Swift Lint check, but I can't find it in the build info. Could you kindly help with this?

File: Maccy/Settings/IgnoreSettingsViewController/IgnoreApplicationsViewController.swift [hunk 1]
```diff
@@ -66,7 +68,8 @@ class IgnoreApplicationsViewController: NSViewController, NSTableViewDataSource,
     if dialog.runModal() == .OK {
       if let appUrl = dialog.url,
          let bundle = Bundle(path: appUrl.path),
-         let bundleIdentifier = bundle.bundleIdentifier {
+         let bundleIdentifier = bundle.bundleIdentifier,
+         !ignoredApps.contains(bundleIdentifier) {
```
[the remark this answers is not in the document]
Comment (phlpsong) [reply Maccy/Settings/IgnoreSettingsViewController/IgnoreApplicationsViewController.swift:71]: I tried but it looks needs to changed more places than the current check statement.

**Comment by phlpsong** [comment]
> > It looks like have an issue with Swift Lint check, but I can't find it in the build info. Could you kindly help with this?
> 
> Not sure why it's not showing in CI logs, but here what I see locally:
> 
> ```
> Maccy/Extensions/Bundle+ApplicationName.swift:16:1: warning: Trailing Newline Violation: Files should have a single trailing newline (trailing_newline)
> ```

Much appreciate it, I left an extra blank line at the bottom of the file. This should be fixed in the latest commit.

**Comment by p0deje** [comment]
Thank you for the contribution!

**Pull request merged**